src/Analyzers/CSharp/Tests/MakeFieldOrPropertyRequired/MakeFieldOrPropertyRequiredTests.cs

```diff
@@ -0,0 +1,606 @@
+        [InlineData("public", "protected internal", "public", true)]
+        [InlineData("public", "protected internal", "private", false)]
+        [InlineData("public", "protected internal", "protected", false)]
+        [InlineData("public", "protected internal", "internal", false)]
+        [InlineData("public", "protected internal", "private protected", false)]
+        [InlineData("public", "protected internal", "protected internal", false)]
+        [InlineData("internal", "public", "public", true)]
+        [InlineData("internal", "public", "private", false)]
+        [InlineData("internal", "public", "protected", false)]
+        [InlineData("internal", "public", "internal", true)]
+        [InlineData("internal", "public", "private protected", false)]
+        [InlineData("internal", "public", "protected internal", true)]
+        [InlineData("internal", "private", "public", true)]
+        [InlineData("internal", "private", "private", false)]
+        [InlineData("internal", "private", "protected", false)]
+        [InlineData("internal", "private", "internal", true)]
+        [InlineData("internal", "private", "private protected", false)]
+        [InlineData("internal", "private", "protected internal", true)]
+        [InlineData("internal", "protected", "public", true)]
+        [InlineData("internal", "protected", "private", false)]
+        [InlineData("internal", "protected", "protected", false)]
+        [InlineData("internal", "protected", "internal", true)]
+        [InlineData("internal", "protected", "private protected", false)]
+        [InlineData("internal", "protected", "protected internal", true)]
+        [InlineData("internal", "internal", "public", true)]
+        [InlineData("internal", "internal", "private", false)]
+        [InlineData("internal", "internal", "protected", false)]
+        [InlineData("internal", "internal", "internal", true)]
+        [InlineData("internal", "internal", "private protected", false)]
+        [InlineData("internal", "internal", "protected internal", true)]
+        [InlineData("internal", "private protected", "public", true)]
+        [InlineData("internal", "private protected", "private", false)]
+        [InlineData("internal", "private protected", "protected", false)]
+        [InlineData("internal", "private protected", "internal", true)]
+        [InlineData("internal", "private protected", "private protected", false)]
+        [InlineData("internal", "private protected", "protected internal", true)]
+        [InlineData("internal", "protected internal", "public", true)]
+        [InlineData("internal", "protected internal", "private", false)]
+        [InlineData("internal", "protected internal", "protected", false)]
+        [InlineData("internal", "protected internal", "internal", true)]
+        [InlineData("internal", "protected internal", "private protected", false)]
+        [InlineData("internal", "protected internal", "protected internal", true)]
```

Comment:
this list looks trhe same as the one above?  if so, extract into common test data segment.